internal/evidence/reactor_test.go

```diff
@@ -284,12 +285,24 @@ func TestReactorBroadcastEvidence(t *testing.T) {
 	}
 
 	rts := setup(ctx, t, stateDBs, 0)
+	// eventBus := eventbus.NewDefault(log.TestingLogger())
```

Comment:
Is this commented-out bit still needed?

Reply:
No its not needed, I will do a final sweep to remove these kind of things, I missed this one.